commons/com.b2international.commons/src/com/b2international/commons/options/OptionsBuilder.java

```diff
@@ -30,10 +30,20 @@
 	OptionsBuilder() {}
 	
 	public OptionsBuilder put(String key, Object value) {
-		options.put(key, value);
+		if (value != null) {
```

Comment:
I don't know if we ever call `put` twice for the same key, but `null` effectively removed the previous value in the method's "before" form, and now it doesn't.